autoload/lightline/bufferline.vim

```diff
@@ -71,14 +72,16 @@ function! s:get_buffer_name(i, buffer, path)
       let l:name = pathshorten(l:name)
     endif
   endif
-  if s:enable_devicons == 1 && exists('*WebDevIconsGetFileTypeSymbol')
+  if exists('*WebDevIconsGetFileTypeSymbol') && s:enable_devicons == 1 && s:reverse_buffer_icons == 0
     let l:name = WebDevIconsGetFileTypeSymbol(fnamemodify(bufname(a:buffer), ':t')) . ' ' . l:name
+  elseif s:reverse_buffer_icons == 1
```

Comment:
If there is only this one condition in the if expression then it will evaluate to true even if devicons are not enabled or installed. Also it will only apply to devicons and not nerdfont icons or nvim_web_devicons.